packages/slate-react/src/components/editable.tsx

```diff
@@ -414,7 +414,6 @@ export const Editable = (props: EditableProps) => {
   const onDOMSelectionChange = useCallback(
     throttle(() => {
       if (
-        !readOnly &&
```

Comment:
@dylans In readOnly mode， if I select a void element, I'll get an error:  "Cannot resolve a Slate point from DOM point" caused by remove this line

Reply:
@Xheldon

Reply:
You may need check which API you use that cause this error, sometimes the selection should be check before you use it, because it can be null sometimes @AsortKeven

Reply:
Ok I’ll look into this tonight.

Reply:
Thanks，I just clone the project，then use the ”metions exmaple“，just change it's Readonly to false，it can reproduce this problem